cmd/buildah/bud.go

```diff
@@ -309,7 +310,7 @@ func budCmd(c *cobra.Command, inputArgs []string, iopts budOptions) error {
 		}
 	}
 
-	options := imagebuildah.BuildOptions{
+	options := define.BuildOptions{
```

Comment:
This shouldn't be necessary.  In this respect, `cmd/buildah` can and should have the same expectations that any of our API consumers have.